src/run.ts

```diff
@@ -38,7 +38,10 @@ const run = async () => {
   const componentDependencies = await Promise.all(config.environments.map(environment => calculateDependencies(environment, components)))
 
   logger.info(`Starting to publish dependency info`)
-  await redisService.write(Object.fromEntries(componentDependencies))
+
+  const data = Object.fromEntries(componentDependencies)
+  await redisService.write({ lastUpdated: new Date().toISOString(), ...data })
```

Comment:
UTC timestamp needs to be included in new Date() function?

Reply:
Good question! apparently it's always UTC

> The toISOString() method of [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) instances returns a string representing this date in the [date time string format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format), a simplified format based on [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), which is always 24 or 27 characters long (YYYY-MM-DDTHH:mm:ss.sssZ or ±YYYYYY-MM-DDTHH:mm:ss.sssZ, respectively). The timezone is always UTC, as denoted by the suffix Z.